data/mobility/county_centers_dim400.json

```diff
@@ -0,0 +1 @@
740335188},{"0":6.7749544315,"1":49.3565035732},{"0":7.2450529898,"1":49.2480906911},{"0":7.0938894873,"1":49.5160280043},{"0":13.3995233004,"1":52.4990949978},{"0":12.5109545077,"1":52.3968652266},{"0":14.3696747877,"1":51.7761737117},{"0":14.4917441916,"1":52.3158789938},{"0":13.0334251358,"1":52.4220291007},{"0":13.7026957512,"1":52.8210154228},{"0":13.8199336057,"1":52.0429179395},{"0":13.4678454642,"1":51.6160855899},{"0":12.629251696,"1":52.6187221515},{"0":14.1512526331,"1":52.6086409559},{"0":13.2102298911,"1":52.9108281588},{"0":13.9493046124,"1":51.6197670362},{"0":14.2118277401,"1":52.245302714},{"0":12.6412115875,"1":52.9898788552},{"0":12.6902680116,"1":52.2432862169},{"0":11.9657802936,"1":53.1133666775},{"0":14.4297213001,"1":51.7555625972},{"0":13.2735890507,"1":52.0707683664},{"0":13.8626391892,"1":53.2055588547},{"0":12.1416516735,"1":54.1488613691},{"0":11.4190128509,"1":53.6158832476},{"0":13.0014290942,"1":53.5420099636},{"0":12.2245416746,"1":53.9120446966},{"0":12.8070589283,"1":54.2118651757},{"0":11.2462766434,"1":53.8208136976},{"0":13.7803695712,"1":53.7861718857},{"0":11.5368692393,"1":53.451917341},{"0":12.9162676793,"1":50.8274408928},{"0":12.9467634985,"1":50.611103845},{"0":13.1367572596,"1":50.9569249775},{"0":12.2336570503,"1":50.4579546794},{"0":12.5264006383,"1":50.7505814552},{"0":13.7932993105,"1":51.0695962421},{"0":14.2311511879,"1":51.2701799305},{"0":14.7535069856,"1":51.2231911489},{"0":13.4838976987,"1":51.2410843386},{"0":13.8790847723,"1":50.9191101584},{"0":12.3792916759,"1":51.3458797475},{"0":12.5939570205,"1":51.2220218247},{"0":12.7919276144,"1":51.4728427251},{"0":12.2236368187,"1":51.8557598377},{"0":11.9745883562,"1":51.4811924965},{"0":11.6417661617,"1":52.11131316},{"0":11.2329202225,"1":52.6792602075},{"0":12.1387221577,"1":51.7987672475},{"0":11.3514962746,"1":52.2200529084},{"0":11.8956726171,"1":51.1442471439},{"0":10.9543694733,"1":51.8214230297},{"0":12.0317481038,"1":52.2669375184},{"0":11.3514163044,"1":51.5330292326},{"0":11.8790112163,"1":51.4324827853},{"0":11.6379244181,"1":51.851277957},{"0":11.8350287492,"1":52.7013601698},{"0":12.6995378625,"1":51.8212575086},{"0":11.0189167263,"1":50.9819431382},{"0":12.08245984,"1":50.8912572335},{"0":11.5901874925,"1":50.9165363342},{"0":10.6874671384,"1":50.6140234704},{"0":11.3179645589,"1":50.9767429584},{"0":10.2560470274,"1":51.3834208596},{"0":10.7322890736,"1":51.4996713865},{"0":10.2150460337,"1":50.8916557381},{"0":10.5478743638,"1":51.1853576065},{"0":10.9780269605,"1":51.3234771734},{"0":10.4048306509,"1":50.6336928377},{"0":10.6951671126,"1":50.9108984979},{"0":11.1542475403,"1":51.1581093648},{"0":10.7381499716,"1":50.4372988278},{"0":10.9689664962,"1":50.7378645291},{"0":11.3650226134,"1":50.9719718617},{"0":11.1371984168,"1":50.4063033701},{"0":11.3029843304,"1":50.631813032},{"0":11.7365465187,"1":50.9037978123},{"0":11.7090318201,"1":50.5800002585},{"0":12.0686099591,"1":50.7433983369},{"0":12.4023769496,"1":50.9499059988}]
```

Comment:
Is it possible to tag the centers with the county ID? Then we would also only need one bigger file and one with the changes with respect to the other.

Reply:
Added the tags in the dataframe. 
I had also considered creating a large file and uploading only the changes for the reduced dimension. However, both files are created from different geojsons and therefore the coordinates are slightly different. Thus, it may be better to upload 2 individual files.